snapcraft/internal/sources/_rpm.py

```diff
@@ -1,6 +1,6 @@
 # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
 #
-# Copyright (C) 2015, 2016 Canonical Ltd
+# Copyright (C) 2015-2016 Canonical Ltd
```

Comment:
Mind adding Neil's copyright here (it got lost with the package creation), and this is for sure 2016 only.